network/src/network.rs

```diff
@@ -217,6 +217,16 @@ impl NetworkState {
         }
     }
 
+    pub(crate) fn remove_peer_store(&self, session_id: SessionId) {
```

Comment:
Remove `info` in `store`, not remove `store`. I think the API name is not suitable.